deploy/prod.yaml

```diff
@@ -86,7 +86,7 @@ nginx-ingress:
     service:
       loadBalancerIP: 35.226.186.40
 
-prometheus-operator:
+kube-prometheus-stack:
```

Comment:
So, do these config values have any effect now? I'd think they'd only have an effect if we're using the `kube-prometheus-stack` helm chart. But IIUC we aren't using that chart? We're using `prometheus` and `grafana` separately?

Reply:
They wouldn't. I hadn't changed GCP values aside from that first commit that changed the umbrella chart, but can do that next. Just wanted the go-ahead from you first.

Reply:
Let me know if there's anything I need to do on the GCP side (IP addresses or anything like that).

Reply:
I should be able to authenticate into the GCP binder. I'll manually deploy staging and get the IP to make sure it all works. Will need an extra PR to plug in the prometheus IP address for prod if we don't want to deploy prod manually (which is probably the case).